container-images/hugo/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos/ruby-24-centos7
```

Comment:
I have a builder image that I have been working on creating a builder image based on alpine. It was inspired by the Hugo and what they use to generate their website [1].  You can see my dockerfile in my branch [2].  Any thoughts about going in this direction?

[1] https://github.com/jojomi/docker-hugo
[2] https://github.com/JaredBurck/uncontained.io/blob/hugo-builder-image/containers/hugo-builder/Dockerfile

Reply:
will merge and continue conversation on idea as a follow